### PR TITLE
Internal JavaScript [Class] of item [Example]

### PR DESCRIPTION
**Pull request by annamalai-saro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -594,6 +594,12 @@ In place of common selectors like class, id or attribute we can use `document.qu
   // Native
   el.matches(selector);
   ```
+<<<<<<< HEAD
```
Comment (camsong) [inline README.md]: These are merge conflicts, can you remove them?

**Comment by annamalai-saro** [comment]
@camsong  I hope now its fine 😄

File: README.md [hunk 2]
```diff
@@ -594,6 +594,11 @@ In place of common selectors like class, id or attribute we can use `document.qu
   // Native
   el.matches(selector);
   ```
+
+- [3.9](#3.9) <a name='3.9'></a> empty
+
+  Removes all child Nodes
+=======
```
Comment (camsong) [inline README.md]: should be removed

**Comment by camsong** [comment]
Close for now, please open a new PR if you are ready

**Pull request closed without merging**